Rollbar/PayloadScrubbing/RollbarPayloadScrubber.cs

```diff
@@ -62,7 +62,7 @@ private RollbarPayloadScrubber()
         public RollbarPayloadScrubber(IEnumerable<string> scrubFields)
         {
             this._payloadFieldNames = 
-                scrubFields.Where(n => !n.Contains('.'))
+                scrubFields.Where(n => !( n.StartsWith(fieldPathRoot) || n.StartsWith(httpRequestBodyPath) || n.StartsWith(httpResponseBodyPath)))
```

Comment:
Won't we never match the last two conditions, since any match would have already matched the first condition?

Also, I was curious about using the `@` string for one of the string literals but not the others. Does this make any difference here?

Reply:
yes, with the currently defined special paths roots only the first one makes the difference (other conditions wouldn't even execute), but i would prefer to keep it as an explicit self-documenting reminder: "None of the Special Paths Roots" to observe.
For currently defined string constants, @ does not make any difference.